src/lib.rs

```diff
@@ -164,6 +164,36 @@
 //! }
 //! ```
 //!
+//! `ReadHandle` is not `Sync` as it is not safe to share a single instance
+//! amongst threads. A fresh `ReadHandle` needs to be created for each thread
+//! either by cloning a `ReadHandle` or from a `ReadHandleFactory`.
```

Comment:
I would add to this:
> The reason for this is that each `ReadHandle` assumes that only one thread operates on it at a time. For details, see the implementation comments on `ReadHandle`. `ReadHandle` _is_ `Send` though, since in order to send a `ReadHandle`, there must be no references to it, so no thread is operating on it.

Reply:
done!

Reply:
words are the same but moved them about a wee bit

Reply:
It would be good to also leave the backtics for code blocks and the **emphasis** on **is** `Send`.

Reply:
done

Reply:
Ah, sorry, I think phrased myself poorly when it came to the backtics. I mean to leave the backtics **in** as well as the emphasis :p

Reply:
ha no worries! I think we're there now?